### PR TITLE
Added tags

### PR DESCRIPTION
**Pull request by LeighamSpringer-Sutton**

### CODE REVIEW

File: 02 Algorithm Reference/02 Initializing Algorithms/02 Setting Cash/content.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<meta name="tag" content="using quantconnect" />
```
Comment (jaredbroad) [inline 02 Algorithm Reference/02 Initializing Algorithms/02 Setting Cash/content.html]: this "Initializing Algorithms" wouldn't be using quantconnect -- its about the API

Comment (LeighamSpringer-Sutton) [reply 02 Algorithm Reference/02 Initializing Algorithms/02 Setting Cash/content.html]: I'm not sure If I'm just not seeing the tag, but I don't see any tag for "API", in the google docs. Do I just put the API tag in anyways ?

Comment (LeighamSpringer-Sutton) [reply 02 Algorithm Reference/02 Initializing Algorithms/02 Setting Cash/content.html]: I'm guessing just remove the tag I have presently and leave this particular html without tags ?

Comment (jaredbroad) [reply 02 Algorithm Reference/02 Initializing Algorithms/02 Setting Cash/content.html]: Just reviewed tags; you're correct there was no tag to describe this section;
Added a new one: "Portfolio Setup" under securities and portfolio;

So you'd need 2 tags:
"portfolio setup" and "securities and portfolio"

**Comment by jaredbroad** [comment]
Awesome;
Every time you push it emails everyone :) Generally only "push" when you're ready for everything to go in. Group your commits together.

**Pull request merged**